### ISSUE TITLE
Filter check boxes  listener ..

### ISSUE THREAD

**Issue by MinaSamirCap** [lapism/SearchBar-SearchView#345]
Do you add any listeners for the check boxes of the filter in search view ..
I need to add two filters but if I check one it un check the  another ..

**Comment by lapism** [comment]
I am not done yet with next release so I will try to fix it in that next release.

**Comment by MinaSamirCap** [comment]
could I contribute with you??
to solve this issue .. or you will work on it ..

**Comment by lapism** [comment]
No, because beta1 branch is totally different with many new stuff.

**Comment by lapism** [comment]
And I will take a look on it.

**Comment by MinaSamirCap** [comment]
okay thanks ..

**Comment by MinaSamirCap** [comment]
I have another problem ..
after adding filters .. for example
`List<SearchFilter> filter = new ArrayList<>();
        filter.add(new SearchFilter("Global", false));
        searchView.setFilters(filter);`

if the user open the search view and press the check box to change its state ..
then I call 
`searchView.getFiltersStates().get(0)`
it always return false .. and it does not return true if the user checked the check box
